views/partials/navbar.handlebars

```diff
@@ -8,6 +8,7 @@
   <div id="navbar-links">
     <a class="underline" href="/discover">Discover</a>
     <a class="underline" href="/dashboard">My Dashboard</a>
-    <a id="login-button">Log In</a>
+    <a id="login-button" href="/auth/google">Log In</a>
+    <a id="logout-button" href="/auth/logout">Log Out</a>
```

Comment:
Wondering: Are we planning on changing visibility of these based on if the user is logged in/out?

Reply:
Sure, added a TODO comment.